Packs/Bitbucket/Integrations/Bitbucket/Bitbucket.py

```diff
@@ -0,0 +1,1479 @@
+    human_readable = []
+    key_list = []
+    for value in results:
+        d = {'Id': value.get('id'),
+             'Title': value.get('title'),
+             'Description': value.get('description'),
+             'SourceBranch': value.get('source', {}).get('branch', {}).get('name'),
+             'DestinationBranch': value.get('destination', {}).get('branch', {}).get('name'),
+             'State': value.get('state'),
+             'CreatedBy': value.get('author', {}).get('display_name'),
+             'CreatedAt': value.get('created_on'),
+             'UpdatedAt': value.get('updated_on')
+             }
+        human_readable.append(d)
+        key_list.append(str(value.get('id')))
+
+    headers = ['Id', 'Title', 'Description', 'SourceBranch', 'DestinationBranch', 'State', 'CreatedBy', 'CreatedAt',
+               'UpdatedAt']
+    readable_output = tableToMarkdown(
+        name=hr_title,
+        t=human_readable,
+        removeNull=True,
+        headers=headers
+    )
+    return CommandResults(
+        readable_output=readable_output,
+        outputs_prefix='Bitbucket.PullRequest',
+        outputs=results,
+        raw_response=results,
+        outputs_key_field=key_list
+    )
+
+
+def issue_comment_create_command(client: Client, args: Dict) -> CommandResults:
+    """ Creates a comment on a specific issue in Bitbucket.
+    Args:
+        client: A Bitbucket client.
+        args: Demisto args.
+    Returns:
+        A CommandResult object with a dictionary contains information about .
+        If a state is provided than the list will contain only PR with the wanted status.
+        If a state is not provided, by default a list of the open pull requests will return.
+    """
+    repo = args.get('repo', client.repository)
+    issue_id = args.get('issue_id', '')
+    content = args.get('content', '')
+    body = {
+        "content": {
+            "raw": content
+        }
+    }
+    response = client.issue_comment_create_request(repo, issue_id, body)
+    return CommandResults(readable_output='The comment was created successfully',
+                          outputs_prefix='Bitbucket.IssueComment',
+                          outputs=response,
+                          raw_response=response,
+                          outputs_key_field=str(response.get('id')))
+
+
+def issue_comment_delete_command(client: Client, args: Dict) -> CommandResults:
+    """ Deletes a comment on a specific issue in Bitbucket.
+    Args:
+        client: A Bitbucket client.
+        args: Demisto args.
+    Returns:
+        A CommandResult object with a success message, in case of a successful deletion.
+    """
+    repo = args.get('repo', client.repository)
+    issue_id = args.get('issue_id', '')
+    comment_id = args.get('comment_id', '')
+    client.issue_comment_delete_request(repo, issue_id, comment_id)
+    return CommandResults(
+        readable_output='The comment was deleted successfully',
```

Comment:
If nothing returns to context, return only the human readable, no need for the rest to appear